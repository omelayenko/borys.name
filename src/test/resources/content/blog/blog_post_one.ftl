Here i tell how to do this ${pathToStory}