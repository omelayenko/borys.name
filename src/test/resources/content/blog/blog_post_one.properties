title=Blog One
description=Blog One Descr