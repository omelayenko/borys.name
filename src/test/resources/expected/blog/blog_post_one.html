<!DOCTYPE html>
<html>
<title>Blog: Blog One</title>
<body color="blue">
<h1>Blog One Descr</h1>Here i tell how to do this blog/blog_post_one</body>
</html>